arches/app/media/js/views/resource/report.js

```diff
@@ -10,7 +10,8 @@ require([
             BaseManagerView.prototype.initialize.call(this, options);
             
             if (location.search.indexOf('print') > 0) {
-                this.viewModel.loading(true);
+                var self = this 
```

Comment:
Could you change `var` to `const` and add a semicolon?

Reply:
Request change made - thanks for flagging that up!